### ISSUE TITLE
fontColor property doesn't work

### ISSUE THREAD

**Issue by AbdulQaharRahmani** [stefcameron/text-to-canvas#64]
The fontColor property does not seem to work properly. When setting the fontColor property with valid color values, such as named colors ('red') or hexadecimal color codes ('#eeff44'), the text color remains unchanged.

```
drawText(
          ctx,
          'this is a sample text for multiline checking',
          {
            x: j * columnWidth + padding,
            y: i * rowHeight + padding,
            width: columnWidth - 2 * padding,
            height: rowHeight - 2 * padding,
            fontSize: 14,
            align: "center",
            vAlign: "top",
            overflow: false,
            fontColor: 'red',
          }
 );
```
fontColor doesn't work when used this way. 
WorkAround: 
the only way that font color can be applied is if it's used like below.
```
drawText(
            ctx,
           [{ text: 'this is a sample text for multiline checking', format: { fontWeight: 'bold', fontColor: 'red' } }],
            {
              x: j * columnWidth + padding,
              y: i * rowHeight + padding,
              width: columnWidth - 2 * padding,
              height: rowHeight - 2 * padding,
              fontSize: 14,
              align: "center",
              vAlign: "top",
              overflow: false,
            }
 )
```
but then the multiline feature no longer works.

**Comment by stefcameron** [comment]
@AbdulQaharRahmani Thank you for reporting this! There is indeed a small bug in the code that doesn't use the `fontColor` specified in `drawText()` options as part of the base font format when rendering the text.

I will get this fixed ASAP.

**Comment by stefcameron** [comment]
Will publish shortly.

**Comment by stefcameron** [comment]
Fixed in v1.1.2

Please LMK if it works for you.

**Comment by AbdulQaharRahmani** [comment]
@stefcameron thanks. the problem is solved.
